lib/order.rb

```diff
@@ -0,0 +1,50 @@
+class Order 
+
+	attr_reader :menu, :order
+	attr_accessor :restaurant
+
+	def initialize
+		@order_list = {}
+		@food = Menu.new.food 	# need to remove dependencies
```

Comment:
You might want this dependency on Menu to be inject in, rather than declared inline.